run-in-container.sh

```diff
@@ -14,5 +14,6 @@ podman run --rm --privileged \
     -e GIT_COMMITTER_NAME="$GIT_USER_NAME" \
     -e GIT_AUTHOR_EMAIL="$GIT_USER_EMAIL" \
     -e GIT_COMMITTER_EMAIL="$GIT_USER_EMAIL" \
+    --entrypoint=python3 \
```

Comment:
Doesn't this mean `run-in-container.sh` won't run f-e-d-c anymore, unless its script is manually passed on cli?

Reply:
You're right, I somehow missed that `run-in-container.sh` was also used for just running f-e-d-c. I've changed it so it selects `python3` as the entrypoint if `python3` happens to be the first argument.